### PR TITLE
Fixup R2 getting started compat date

### PR DESCRIPTION
**Pull request by vlovich**
Also add note about compat flag.

### CODE REVIEW

File: content/r2/get-started.md [hunk 1]
```diff
@@ -106,11 +106,12 @@ Find your Account ID by logging in to the Cloudflare dashboard > **Overview** >
 ```toml
 name = "<YOUR_WORKER_NAME>"
 type = "javascript"
-compatibility_date = "2022-02-10"
+compatibility_date = "2022-04-18"
 
 account_id = "YOUR_ACCOUNT_ID" # ← Replace with your Account ID.
 workers_dev = true
 ```
+If you need to use an older compatibility date, you currently need to enable the `r2_public_beta_bindings` compatability flag.
```
Comment (deadlypants1973) [inline content/r2/get-started.md]:
```suggestion
If you need to use an older compatibility date, you currently need to enable the `r2_public_beta_bindings` [compatability flag](https://developers.cloudflare.com/workers/platform/compatibility-dates/).
```

File: content/r2/get-started.md [hunk 2]
```diff
@@ -106,11 +106,12 @@ Find your Account ID by logging in to the Cloudflare dashboard > **Overview** >
 ```toml
 name = "<YOUR_WORKER_NAME>"
 type = "javascript"
-compatibility_date = "2022-02-10"
+compatibility_date = "2022-04-18"
 
 account_id = "YOUR_ACCOUNT_ID" # ← Replace with your Account ID.
 workers_dev = true
 ```
+If you need to use an older compatibility date, you currently need to enable the `r2_public_beta_bindings` [compatability flag](/workers/platform/compatability-dates).
```
Comment (deadlypants1973) [inline content/r2/get-started.md]:
```suggestion
If you need to use an older compatibility date, you currently need to enable the `r2_public_beta_bindings` [compatibility flag](/workers/platform/compatability-dates).
```

Comment (pedrosousa) [reply content/r2/get-started.md]: We should also adjust the link URL to `/workers/platform/compatibility-dates/` (fix typo, add end slash).

File: content/r2/get-started.md [hunk 3]
```diff
@@ -106,11 +106,12 @@ Find your Account ID by logging in to the Cloudflare dashboard > **Overview** >
 ```toml
 name = "<YOUR_WORKER_NAME>"
 type = "javascript"
-compatibility_date = "2022-02-10"
+compatibility_date = "2022-04-18"
 
 account_id = "YOUR_ACCOUNT_ID" # ← Replace with your Account ID.
 workers_dev = true
 ```
+If you need to use an older compatibility date, you currently need to enable the `r2_public_beta_bindings` [compatibility flag](/workers/platform/compatability-dates).
```
Comment (deadlypants1973) [inline content/r2/get-started.md]:
```suggestion
If you need to use an older compatibility date, you need to enable the `r2_public_beta_bindings` [compatibility flag](/workers/platform/compatability-dates).
```

File: content/r2/get-started.md [hunk 4]
```diff
@@ -106,11 +106,12 @@ Find your Account ID by logging in to the Cloudflare dashboard > **Overview** >
 ```toml
 name = "<YOUR_WORKER_NAME>"
 type = "javascript"
-compatibility_date = "2022-02-10"
+compatibility_date = "2022-04-18"
 
 account_id = "YOUR_ACCOUNT_ID" # ← Replace with your Account ID.
 workers_dev = true
 ```
+If you need to use an older compatibility date, you need to enable the `r2_public_beta_bindings` [compatibility flag](/workers/platform/compatability-dates).
```
Comment (lukeed) [inline content/r2/get-started.md]:
```suggestion
If you need to use an older compatibility date, you need to enable the `r2_public_beta_bindings` [compatibility flag](/workers/platform/compatibility-dates/).
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**